pagopa/io-app-design-system
# [IOPLT-112] Adds the FooterWithButtons component

## Short description
This PR creates a new version of the footer with buttons with the new version of buttons from DS library

## How to test
Check the specific page in the Example app to see how it changes.

<img src="https://github.com/pagopa/io-app-design-system/assets/3959405/ac5f85b6-50d9-4001-94b8-7b604541199d" height="600"/>
